addons/binding/org.openhab.binding.s7/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.binding</groupId>
+    <artifactId>pom</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
Correct me if I'm wrong but I think I did rebase my branch yesterday : https://github.com/openhab/openhab2-addons/pull/1606/commits/ee95e243a71360143cf29dc36ee8ca6006aacc5f

Actually, I just figured out, I had to `mvn licence:format` on the repo root folder not on my bundle folder. I think it's now ok.